src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,29 @@
      * Андрей - 600
      * София - 900</p>
      */
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
+        if (startDate.isAfter(endDate)) {
+            throw new IllegalDateParametersException();
```

Comment:
add message to exception(you can find it in description)

Reply:
done